ospd_openvas/daemon.py

```diff
@@ -229,6 +230,17 @@
 OID_SMB_AUTH = "1.3.6.1.4.1.25623.1.0.90023"
 OID_ESXI_AUTH = "1.3.6.1.4.1.25623.1.0.105058"
 OID_SNMP_AUTH = "1.3.6.1.4.1.25623.1.0.105076"
+OID_PING_HOST = "1.3.6.1.4.1.25623.1.0.100315"
+
+
+class AliveTest(IntEnum):
+    """ Alive Tests. """
+
+    ALIVE_TEST_TCP_ACK_SERVICE = (1,)
+    ALIVE_TEST_ICMP = (2,)
+    ALIVE_TEST_ARP = (4,)
+    ALIVE_TEST_CONSIDER_ALIVE = (8,)
+    ALIVE_TEST_TCP_SYN_SERVICE = 16
```

Comment:
Why are the all values besides ALIVE_TEST_TCP_SYN_SERVICE tuples and ALIVE_TEST_TCP_SYN_SERVICE not?

Reply:
Not sure what happened here. Maybe autohooks? They should be all integer. Fixed.